File: LP/GO/ex4.go
package main

import(
	"fmt"
)

func main(){
	var(
		tempF float32
		tempC float32
	)

	fmt.Print("Digite uma temperatura em Fahrentheit: ")
	fmt.Scan(&tempF)

	tempC = (tempF - 32) * 5 / 9

	fmt.Printf("%.1f°F equivalem a %.1f°C.\n", tempF, tempC)
}

File: 2ºAno/LP/C/ex4.c
#include <stdio.h>
#include <string.h>

void main() {
    int tempf;
    int tempc;

    printf("Digite a temperatura em Fahrenheit: ");
    scanf("%d", &tempf);
    tempc = (tempf - 32) * 5 / 9;
    printf("A temperatura em Celsius é: %d\n", tempc);
}